src/pages/publication/MyPublications.tsx

```diff
@@ -11,22 +17,36 @@ export interface DummyPublicationListElement {
   status: string;
 }
 
+const StyledSection = styled.section`
+  text-align: center;
+`;
+
```

Comment:
synes det er rart å ha hele publication list inni en section. 
`Do not use the <section> element as a generic container; this is what <div> is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section